blockchain/indexers/spendconsumer_test.go

```diff
@@ -16,19 +15,14 @@ import (
 
 // TestSpendConsumer ensures the spend consumer behaves as expected.
 func TestSpendConsumer(t *testing.T) {
-	dbPath, err := os.MkdirTemp("", "test_spendconsumer")
-	if err != nil {
-		t.Fatalf("unable to create test db path: %v", err)
-	}
+	dbPath := t.TempDir()
 
 	db, err := database.Create("ffldb", dbPath, wire.SimNet)
 	if err != nil {
-		os.RemoveAll(dbPath)
 		t.Fatalf("error creating db: %v", err)
 	}
 	defer func() {
```

Comment:
You could just call `defer db.Close()` directly now if you want since there is no longer a second func to call.